test/MutationDataTest.py

```diff
@@ -123,6 +123,13 @@ def testPickleable(self):
         import cPickle
         cPickle.dump(m, open("out/testMDPickle.pkl", 'w'))
 
+    def test_copy(self):
+        """Test annotation copy """
```

Comment:
This does that already.  ``getAnnotation()`` does not return the value.  It returns an instance of the Annotation class.  Then it is tested for equality.  See ``Annotation.py`` for more info -- it has a custom ``__eq__`` method.